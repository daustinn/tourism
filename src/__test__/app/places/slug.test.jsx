// generate test facke

import PlaceSlug from 'app/places/[slug]/loading'
import { render } from '@testing-library/react'

describe('Places Slug', () => {
  it('should render the slug', () => {
    const { container } = render(<PlaceSlug />)
    expect(container).toMatchSnapshot()
  })
})
