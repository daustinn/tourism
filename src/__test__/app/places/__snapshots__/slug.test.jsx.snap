// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Places Slug should render the slug 1`] = `
<div>
  <div
    class="max-w-5xl mx-auto font-sans w-full py-6 px-4"
  >
    <div
      class="flex flex-col"
    >
      <div
        class="h-10 rounded-xl animate-pulse flex bg-neutral-200 w-[200px] items-center"
      />
      <div
        class="space-y-3 mt-5"
      >
        <div
          class="w-full rounded-2xl overflow-hidden"
        >
          <div
            class="grid min-h-[30svh] grid-cols-4 gap-2 max-md:grid-cols-2"
          >
            <div
              class="aspect-[9/9] overflow-hidden group bg-neutral-200 animate-pulse
                    col-span-2 row-span-2
                 "
            />
            <div
              class="aspect-[9/9] overflow-hidden group bg-neutral-200 animate-pulse
                    
                 "
            />
            <div
              class="aspect-[9/9] overflow-hidden group bg-neutral-200 animate-pulse
                    
                 "
            />
            <div
              class="aspect-[9/9] overflow-hidden group bg-neutral-200 animate-pulse
                    
                 "
            />
            <div
              class="aspect-[9/9] overflow-hidden group bg-neutral-200 animate-pulse
                    
                 "
            />
          </div>
        </div>
        <div
          class="w-32 h-5 bg-neutral-200 rounded-xl animate-pulse"
        />
      </div>
    </div>
  </div>
</div>
`;
